test/catalogd-e2e/unpack_test.go

```diff
@@ -14,6 +14,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 
 	catalogdv1 "github.com/operator-framework/operator-controller/catalogd/api/v1"
+	"github.com/operator-framework/operator-controller/test/utils"
```

Comment:
nitpick : can we import this as testutil? because we already have internal/util in code.